src/outlookCalendar/getOutlookEvents.ts

```
@@ -172,10 +199,19 @@ export const sanitizeExchangeUrl = (serverUrl: string): string => {
 
     return sanitizedUrl;
   } catch (error) {
+    const errorMessage = error instanceof Error ? error.message : String(error);
+
+    // Re-throw validation errors without attempting fallback, except for cases
+    // where the URL looks like it's missing a protocol (e.g., "domain.com:443")
+    // Match IPv4 addresses or hostnames with dots followed by port (not scheme:port like "ftp:21")
+    if (errorMessage.includes('Invalid protocol') && !serverUrl.match(/^(?:\d{1,3}(?:\.\d{1,3}){3}|[a-z0-9-]+\.[a-z0-9.-]+):\d+/i)) {
```

ESLint

[error] Replace `errorMessage.includes('Invalid·protocol')·&&·!serverUrl.match(/^(?:\d{1,3}(?:\.\d{1,3}){3}|[a-z0-9-]+\.[a-z0-9.-]+):\d+/i)` with `⏎······errorMessage.includes('Invalid·protocol')·&&⏎······!serverUrl.match(⏎········/^(?:\d{1,3}(?:\.\d{1,3}){3}|[a-z0-9-]+\.[a-z0-9.-]+):\d+/i⏎······)⏎····`

(prettier/prettier)